modules/git/git_commits/README.md

```diff
@@ -0,0 +1,9 @@
+This is a module intending to teach first-year engineering students about what a good git committing behaviour looks like, what a good message looks like and why they want to follow that behaviour.
+
+There is a presentation in the [slides-directory](slides/). It has been largely based on the work done by Amanda Berg and Simon Osnes in their bachelor's degree project from 2021. A summary of the relevant parts can be found [here](https://docs.google.com/document/d/1emMWhN-MbPgIkLxLoTEL-98FRPMu-oD9lh0nBosCc9A/edit?usp=sharing). The project in its entirety can be found [here](https://kth.diva-portal.org/smash/record.jsf?dswid=-3489&pid=diva2%3A1599553&c=1&searchType=SIMPLE&language=en&query=Amanda+Berg&af=%5B%5D&aq=%5B%5B%5D%5D&aq2=%5B%5B%5D%5D&aqe=%5B%5D&noOfRows=50&sortOrder=author_sort_asc&sortOrder2=title_sort_asc&onlyFullText=false&sf=all)
+
+A recording of the presentation can be found [here](https://www.youtube.com/watch?v=YSXv6-aXWkk)
+
+A recording of the exercise-video can be found [here](https://youtu.be/-xt_bEMTssU)

```

Comment:
I'm still unsure if it will work with the Java for-loop example. Either 
1. explain it more thoroughly and use a pointer to point at and highlight the relevant parts of the code that you're talking about. Or
2. change it to Python which is much closer to human language.

For example:
```python
def sum_positive(num_list):
    sum_list = 0

    for i in range(1, len(num_list)):
        num = num_list[i]
        if num > 0:
            sum_list += num_list[i]

    return sum_list
```

And then run the code, `sum_positive([1, 2, 3])` to show the error. And then run it with the fixed version too, to show the difference the effect the change had.

I think you can just re-record this short part and then edit the pieces together.

Reply:
Or opt for [the terminal based exercise I suggested above](https://github.com/dbosk/introtools/pull/88#discussion_r941211098). Feels like there might be more re-recording though.

Reply:
Maybe that's the best since they should know the terminal by this time. But they might not have had even the first lecture of a programming course.

Reply:
Any updates @amaberg, we're closing in on course start on Monday! :-D

Reply:
I haven't had any time in the last week. I will record the updated version on Saturday. Hopefully that is enough time